=

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import Image from 'next/image'
+// MUI //
+import Drawer from '@mui/material/Drawer'
+import { Box } from '@mui/system'
+import Collapse from '@mui/material/Collapse'
+// other mui dependencies
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+
+// icons and text
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import StarIcon from '@mui/icons-material/Star'
+import SettingsIcon from '@mui/icons-material/Settings'
+
+export default function NavigationDrawer() {
+  // Params for the Drawer
+  const drawerWidth = 280
+  // Logo definition
+  const logo = (
+    <Image
+      src="/../public/images/CCAHT-Logo.png"
+      alt="CCAHT-logo"
+      height="70"
+      width="239"
+    />
+  )
+  // state handlers for expandable lists in drawer
+  const [open, setOpen] = React.useState(true)
+  const handleClick = () => {
+    setOpen(!open)
+  }
+  // Drawer setup
+  // TODO ICONS
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <Drawer
+        variant="permanent"
+        sx={{
+          width: drawerWidth,
+        }}
+      >
+        <Box>
+          <Box sx={{ pr: 3, mt: 5 }}>{logo}</Box>
```

Comment:
make this prop the following (had to figure this out after some experimentation):
```
sx={{ display: 'flex', px: 2, pt: 2, justifyContent: 'center' }}
```
`px` puts padding on both the left and right side of the container.
`pt` puts the same amount of padding on the top of the container (these two used together will help space the image correct)
`display: 'flex'` makes this container a flex box so that we can use
`justifyContent: 'center'` to finish centering the image.